src/nl/surf/eduhub_rio_mapper/cli.clj

```diff
@@ -197,7 +197,8 @@
                                                       :opleidingseenheidcode)})
 
                    (#{:error :time-out} status)
-                   (assoc :phase (-> data :errors :phase)
+                   (assoc :http-messages (-> data :http-messages)
+                          :phase (-> data :errors :phase)
                           :message (-> data :errors :message))))
```

Comment:
Willen we deze feedback alleen bij fouten? LIjkt me niet handig voor development